packages/next/taskfile.js

```diff
@@ -86,6 +86,8 @@ const externals = {
     'next/dist/build/webpack/plugins/terser-webpack-plugin',
 
   // TODO: Add @swc/helpers to externals once @vercel/ncc switch to swc-loader
+
+  undici: 'undici',
```

Comment:
Any reason to add `undici` here? It will be overwritten by `externals['undici'] = 'next/dist/compiled/undici'` below anyway.

Reply:
I added this to match how `node-fetch` is include (line 70)